exchange/exchange_test.go

```diff
@@ -539,7 +540,7 @@ func TestTwoBiddersDebugDisabledAndEnabled(t *testing.T) {
 
 func TestOverrideWithCustomCurrency(t *testing.T) {
 
-	mockCurrencyClient := &mockCurrencyRatesClient{
+	mockCurrencyClient := &fakeCurrencyRatesHttpClient{
```

Comment:
I thought we favored the `mock` term over the `fake` term. I agree with `fake`

Reply:
[Depends on the context.](https://stackoverflow.com/questions/346372/whats-the-difference-between-faking-mocking-and-stubbing#:~:text=Mock%20%2D%20an%20object%20on%20which,people%20simply%20call%20them%20mocks.)

I personally prefer to use mock when testify mock's framework is used with test defined behavior/expectations and fake when the object does very little and is somewhat configurable by the test, such as with the request body in this instance.